Cauldron/Assets/Scripts/UI/NewGameButton.cs

```diff
@@ -0,0 +1,29 @@
+﻿using UnityEngine;
+using Zenject;
+
+public class NewGameButton : MonoBehaviour
+{
+    [SerializeField] private GameObject startGameText;
```

Comment:
For your information: you can put strings here and mark them with [Localize], they will be changed auto-magically by key NewGameButton.startGameText (see GameFXManager, for example).

You also need to press Collect Ui in the menu scope localization tool to serialize all localization entities, but I guess you figured that out)